float64 probability
int16 min_x 
int16 min_y
int16 max_x
int16 max_y
int8 label 